### ISSUE TITLE
Windows got errpr: RuntimeError: Library cublas64_11.dll is not found or cannot be loaded

### ISSUE THREAD

**Issue by lucasjinreal** [guillaumekln/faster-whisper#535]
RuntimeError: Library cublas64_11.dll is not found or cannot be loaded

I have installed CUDA and cudnn, the torch loads and use cuda normally.

**Comment by ChihTsungLu** [comment]
I got the same error .

**Comment by lucasjinreal** [comment]
Looks like CUDA12 not supported. But install ctranslate2 didn't have a fully automatically building script for users pip install auto build on target lib.

**Comment by phineas-pta** [comment]
if u want cuda 12 must build ctranslate2 from source

**Comment by benjaminchensz** [comment]
> RuntimeError: Library cublas64_11.dll is not found or cannot be loaded
> 
> I have installed CUDA and cudnn, the torch loads and use cuda normally.

See if you can find the cublas64_12.dll and rename it to cublas64_11.dll.

**Comment by BBC-Esq** [comment]
Or you could just install CUDA 11.8 unless you truly know the difference and can't live without CUDA 12.1?

**Comment by FreeTymeKiyan** [comment]
Is this really a cuda 12 issue? 
I was trying to download cublas from https://developer.nvidia.com/hpc-sdk-downloads but for windows it says
> The NVIDIA HPC SDK for Windows will be available at a later date.

It seems to me that windows is just not supported officially. Unless there's some hack that I don't know of.

**Comment by phineas-pta** [comment]
cublas is included in cuda

**Comment by FreeTymeKiyan** [comment]
@phineas-pta yeah I actually found the file, `cublas64_11.dll`, in the CUDA 11.8. I also made sure that the directory is in PATH. 

But for some reason it still throws the same error

**Comment by FreeTymeKiyan** [comment]
My issue was resolved after installing CUDA 11.8 and made sure that 

1. `nvcc --version` shows `Cuda compilation tools, release 11.8, V11.8.89`
2. my vscode is completely restarted (the terminal wasn't able to pick up the environment variable changes)

**Comment by lucasjinreal** [comment]
Using insame-faster-whisper, that lib faster than this one.

**Comment by BBC-Esq** [comment]
I just got a response on the ctranslate2 github "issue" they say that CUDA 12+ is supported if you compile from source, but my question was whether it's possible to have it supported out-of-the-box without having to compile...Anyways, see my prior "issue" on this github regarding "insane" faster whisper and the differences between it and faster-whisper - spoiler, faster-whisper is still faster with everything equal...it just doesn't support the "batching" like "insane" does.

Regarding the CUDA 12 issue, you have to use CUDA 11.8 currently if you pip install faster-whisper (and hence ctranslate2)...unfortunately but hopefully they'll update with CUDA support out-of-the-box as newer PyTorch versions come out.

**Comment by RazeBerry** [comment]
> > > RuntimeError: Library cublas64_11.dll is not found or cannot be loaded
> > > I have installed CUDA and cudnn, the torch loads and use cuda normally.
> > 
> > 
> > See if you can find the cublas64_12.dll and rename it to cublas64_11.dll.
> 
> This solved my case. but I did copy&rename in case I forget I did this.
> 
> The file locaton was C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.3\bin\

For the rest of us who have CUDA 11, how is this supposed to work? Also suffering from the same problem

**Comment by kofeisen** [comment]
RuntimeError: Library cublas64_12.dll is not found or cannot be loaded.
'copy and rename' Solve the problem

**Comment by lukeewin** [comment]
> RuntimeError: Library cublas64_12.dll is not found or cannot be loaded. 'copy and rename' Solve the problem

I encountered the same issue.
Are you renaming cublas64_11.dll to cublas64_12.dll?

**Comment by kofeisen** [comment]
> > RuntimeError: Library cublas64_12.dll is not found or cannot be loaded. 'copy and rename' Solve the problem
> 
> I encountered the same issue. Are you renaming cublas64_11.dll to cublas64_12.dll?

Yes.

**Comment by gzqccnu** [comment]
>just go to https://github.com/Purfview/whisper-standalone-win/releases/tag/libs download the `the corresponding ZIP file`
>lack ``cublas64_11.dll``，download **cuBLAS.and.cuDNN_CUDA11_win_v2.zip** and add it to your cuda bin

**Comment by pookiepied** [comment]
Bruh, renaming the file is working. LOL
